pydra/engine/core.py

```diff
@@ -182,6 +182,7 @@ def __init__(
 
         self.plugin = None
         self.hooks = TaskHook()
+        self.errored = False
```

Comment:
should this be a property. i don't think you want a user to set this.

Reply:
you're probably right, but I'm using this in the Submitter

Reply:
oh, you mean I should create a property?